5.5/examples/replica/mysql_replica.json

```diff
@@ -68,6 +68,23 @@
       }
     },
     {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "mysql-slave"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "512Mi"
```

Comment:
it's meant to be kept in alignment with the OSO templates where-ever possible, there's no good reason *not* to do so.  (So that means changing the master also.  They should probably both use the same parameter since it would make no sense to have different sized volumes for replicas of the same database).